stringview.js

```diff
@@ -0,0 +1,655 @@
+/*\
+|*|
+|*|  StringView - Mozilla Developer Network
+|*|
+|*|  Revision #8, October 6, 2014
+|*|
+|*|  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/StringView
+|*|  https://developer.mozilla.org/en-US/docs/User:fusionchess
+|*|
+|*|  This framework is released under the GNU Lesser General Public License, version 3 or later.
```

Comment:
That's an unacceptable license, by the way.

I'm not taking StringView. There's a whiteout-encoding package (I forget the name) that I've used in other libraries for importing TextEncoder/TextDecoder in the poor environments that don't have them.